stories/topics/proc-gcp-generate-variables.adoc

```diff
@@ -0,0 +1,54 @@
+[id="proc-gcp-generate-variables"]
+
+= Generating data files by running the ansible-on-clouds-ops container
+
+The following commands generate the required data file. 
+These commands create a directory, and  an empty data template that, when populated is used TBD. 
+
+.Procedure
+. Create a folder to hold the configuration files.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+$ mkdir command_generator_data
+----
+. Populate the `command_generator_data` folder with the configuration file template.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+$ docker run --rm -v $(pwd)/command_generator_data:/data $IMAGE \
+command_generator_vars aws_remove_extension_nodes \
```

Comment:
From
command_generator_vars aws_remove_extension_nodes \
To
command_generator_vars gcp_setup_logging_monitoring \